.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+- 'node'
+branches:
```

Comment:
Why only for master branch?

Reply:
This was only the initial configuration file the idea is use the develop branch as well in the blacklisted branch for running the build.

https://docs.travis-ci.com/user/customizing-the-build/#building-specific-branches

As team we can review all the configurations to be sure we are using the correct ones for the project